autosklearn/estimators.py

```diff
@@ -253,11 +253,11 @@ def fit(self, *args, **kwargs):
         self._automl = self.build_automl()
         super(AutoSklearnEstimator, self).fit(*args, **kwargs)
 
-    def fit_ensemble(self, task=None, metric=None, precision='32',
+    def fit_ensemble(self, y, task=None, metric=None, precision='32',
```

Comment:
Well, I'd say rather not - we don't need to change the signature for now for any other method in AutoML and Regressor